### PR TITLE
Update ClientRuntime across all projects

### PR DESCRIPTION
**Pull request by cormacpayne**
## Description

Update `Microsoft.Rest.ClientRuntime` and `Microsoft.Rest.ClientRuntime.Azure` to the latest versions across all projects.

## Checklist

- [x] I have read the [_Submitting Changes_](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate [change log has been updated](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#updating-the-change-log)
- [x] The PR does not introduce [breaking changes](https://github.com/Azure/azure-powershell/blob/preview/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] the changes have gone through a [cmdlet design review](https://github.com/Azure/azure-powershell-cmdlet-review-pr)
    - [ ] the cmdlet markdown files were [generated using the `platyPS` module](https://github.com/Azure/azure-powershell/blob/preview/documentation/development-docs/help-generation.md)

### CODE REVIEW

**Comment by maddieclayton** [comment]
sign job here: https://azuresdkci.westus2.cloudapp.azure.com/view/PowerShell/job/ps-sign/906/

**Comment by cormacpayne** [comment]
Sign job: https://azuresdkci.westus2.cloudapp.azure.com/view/PowerShell/job/ps-sign/909/

File: src/ResourceManager/Automation/Commands.Automation.Test/Commands.Automation.Test.csproj [hunk 1]
```diff
@@ -85,10 +85,6 @@
     <Reference Include="Microsoft.IdentityModel.Clients.ActiveDirectory.WindowsForms, Version=2.28.3.860, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
       <HintPath>..\..\..\packages\Microsoft.IdentityModel.Clients.ActiveDirectory.2.28.3\lib\net45\Microsoft.IdentityModel.Clients.ActiveDirectory.WindowsForms.dll</HintPath>
     </Reference>
-    <Reference Include="Microsoft.Rest.ClientRuntime, Version=2.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
```
Comment (markcowl) [inline src/ResourceManager/Automation/Commands.Automation.Test/Commands.Automation.Test.csproj]: How can this be removed?  It is required by the auth library below

File: src/ResourceManager/Compute/Commands.Compute.Test/packages.config [hunk 2]
```diff
@@ -18,8 +18,6 @@
   <package id="Microsoft.Bcl.Build" version="1.0.14" targetFramework="net45" />
   <package id="Microsoft.IdentityModel.Clients.ActiveDirectory" version="2.28.3" targetFramework="net45" />
   <package id="Microsoft.Net.Http" version="2.2.28" targetFramework="net45" />
-  <package id="Microsoft.Rest.ClientRuntime" version="2.3.11" targetFramework="net452" />
```
Comment (markcowl) [inline src/ResourceManager/Compute/Commands.Compute.Test/packages.config]: ??

File: src/ResourceManager/DataFactories/Commands.DataFactories.Test/Commands.DataFactories.Test.csproj [hunk 3]
```diff
@@ -102,10 +102,6 @@
       <SpecificVersion>False</SpecificVersion>
       <HintPath>..\..\..\packages\Microsoft.IdentityModel.Clients.ActiveDirectory.2.28.3\lib\net45\Microsoft.IdentityModel.Clients.ActiveDirectory.WindowsForms.dll</HintPath>
     </Reference>
-    <Reference Include="Microsoft.Rest.ClientRuntime, Version=2.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
```
Comment (markcowl) [inline src/ResourceManager/DataFactories/Commands.DataFactories.Test/Commands.DataFactories.Test.csproj]: ??

File: src/ResourceManager/DataMigration/Commands.DataMigration/ChangeLog.md [hunk 4]
```diff
@@ -18,9 +18,14 @@
         - Additional information about change #1
 -->
 ## Current Release
+<<<<<<< HEAD
```
Comment (markcowl) [inline src/ResourceManager/DataMigration/Commands.DataMigration/ChangeLog.md]: merge issue

File: src/ResourceManager/DevSpaces/Commands.DevSpaces/ChangeLog.md [hunk 5]
```diff
@@ -18,9 +18,14 @@
         - Additional information about change #1
 -->
 ## Current Release
+<<<<<<< HEAD
```
Comment (markcowl) [inline src/ResourceManager/DevSpaces/Commands.DevSpaces/ChangeLog.md]: merge issue

File: src/ResourceManager/DevTestLabs/Commands.DevTestLabs/ChangeLog.md [hunk 6]
```diff
@@ -18,9 +18,14 @@
         - Additional information about change #1
 -->
 ## Current Release
+<<<<<<< HEAD
```
Comment (markcowl) [inline src/ResourceManager/DevTestLabs/Commands.DevTestLabs/ChangeLog.md]: merge issue

File: src/ResourceManager/Dns/Commands.Dns/ChangeLog.md [hunk 7]
```diff
@@ -18,9 +18,14 @@
         - Additional information about change #1
 -->
 ## Current Release
+<<<<<<< HEAD
```
Comment (markcowl) [inline src/ResourceManager/Dns/Commands.Dns/ChangeLog.md]: merge issue

File: src/ResourceManager/Dns/Commands.Dns.Test/packages.config [hunk 8]
```diff
@@ -16,8 +16,6 @@
   <package id="Microsoft.Bcl.Build" version="1.0.14" targetFramework="net45" />
   <package id="Microsoft.IdentityModel.Clients.ActiveDirectory" version="2.28.3" targetFramework="net45" />
   <package id="Microsoft.Net.Http" version="2.2.28" targetFramework="net45" />
-  <package id="Microsoft.Rest.ClientRuntime" version="2.3.11" targetFramework="net452" />
```
Comment (markcowl) [inline src/ResourceManager/Dns/Commands.Dns.Test/packages.config]: ??

File: src/ResourceManager/EventGrid/Commands.EventGrid/ChangeLog.md [hunk 9]
```diff
@@ -18,9 +18,14 @@
         - Additional information about change #1
 -->
 ## Current Release
+<<<<<<< HEAD
```
Comment (markcowl) [inline src/ResourceManager/EventGrid/Commands.EventGrid/ChangeLog.md]: merge issue

File: src/Common/Commands.Common.Aks/Commands.Common.Aks.csproj [hunk 10]
```diff
@@ -40,11 +40,11 @@
   </PropertyGroup>
   <ItemGroup>
     <Reference Include="Microsoft.Rest.ClientRuntime, Version=2.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
-      <HintPath>..\..\packages\Microsoft.Rest.ClientRuntime.2.3.11\lib\net452\Microsoft.Rest.ClientRuntime.dll</HintPath>
+      <HintPath>..\..\packages\Microsoft.Rest.ClientRuntime.2.3.12\lib\net452\Microsoft.Rest.ClientRuntime.dll</HintPath>
```
Comment (markcowl) [inline src/Common/Commands.Common.Aks/Commands.Common.Aks.csproj]: So, below we remove these from the tests, can we consistently depend on common dependencies for this, and remove these references everywhere?

File: src/ResourceManager/Relay/Commands.Relay/ChangeLog.md [hunk 11]
```diff
@@ -18,9 +18,14 @@
         - Additional information about change #1
 -->
 ## Current Release
+<<<<<<< HEAD
```
Comment (markcowl) [inline src/ResourceManager/Relay/Commands.Relay/ChangeLog.md]: merge issue

File: src/ResourceManager/Reservations/Commands.Reservations/ChangeLog.md [hunk 12]
```diff
@@ -19,9 +19,14 @@
         - Additional information about change #1
 -->
 ## Current Release
+<<<<<<< HEAD
```
Comment (markcowl) [inline src/ResourceManager/Reservations/Commands.Reservations/ChangeLog.md]: merge issue

File: src/ResourceManager/Scheduler/Commands.Scheduler/ChangeLog.md [hunk 13]
```diff
@@ -18,9 +18,14 @@
         - Additional information about change #1
 -->
 ## Current Release
+<<<<<<< HEAD
```
Comment (markcowl) [inline src/ResourceManager/Scheduler/Commands.Scheduler/ChangeLog.md]: same

File: src/ResourceManager/ServiceFabric/Commands.ServiceFabric/ChangeLog.md [hunk 14]
```diff
@@ -18,9 +18,14 @@
         - Additional information about change #1
 -->
 ## Current Release
+<<<<<<< HEAD
```
Comment (markcowl) [inline src/ResourceManager/ServiceFabric/Commands.ServiceFabric/ChangeLog.md]: same

File: src/ResourceManager/SignalR/Commands.SignalR/ChangeLog.md [hunk 15]
```diff
@@ -18,9 +18,14 @@
         - Additional information about change #1
 -->
 ## Current Release
+<<<<<<< HEAD
```
Comment (markcowl) [inline src/ResourceManager/SignalR/Commands.SignalR/ChangeLog.md]: same

File: src/ResourceManager/SignalR/Commands.SignalR.Test/packages.config [hunk 16]
```diff
@@ -14,8 +14,6 @@
   <package id="Microsoft.Bcl.Build" version="1.0.14" targetFramework="net452" />
   <package id="Microsoft.IdentityModel.Clients.ActiveDirectory" version="2.28.3" targetFramework="net452" />
   <package id="Microsoft.Net.Http" version="2.2.28" targetFramework="net452" />
-  <package id="Microsoft.Rest.ClientRuntime" version="2.3.11" targetFramework="net452" />
```
Comment (markcowl) [inline src/ResourceManager/SignalR/Commands.SignalR.Test/packages.config]: see earlier comment - I think we should do this everywhere that uses common dependencies, or nowhere

File: src/ResourceManager/Storage/Commands.Management.Storage/ChangeLog.md [hunk 17]
```diff
@@ -18,9 +18,14 @@
         - Additional information about change #1
 -->
 ## Current Release
+<<<<<<< HEAD
```
Comment (markcowl) [inline src/ResourceManager/Storage/Commands.Management.Storage/ChangeLog.md]: merge issue

File: src/ResourceManager/StreamAnalytics/Commands.StreamAnalytics/ChangeLog.md [hunk 18]
```diff
@@ -18,9 +18,14 @@
         - Additional information about change #1
 -->
 ## Current Release
+<<<<<<< HEAD
```
Comment (markcowl) [inline src/ResourceManager/StreamAnalytics/Commands.StreamAnalytics/ChangeLog.md]: merge issue

File: src/ResourceManager/Subscription/Commands.Subscription/ChangeLog.md [hunk 19]
```diff
@@ -18,9 +18,14 @@
         - Additional information about change #1
 -->
 ## Current Release
+<<<<<<< HEAD
```
Comment (markcowl) [inline src/ResourceManager/Subscription/Commands.Subscription/ChangeLog.md]: merge issue

File: src/ResourceManager/TrafficManager/Commands.TrafficManager2/ChangeLog.md [hunk 20]
```diff
@@ -18,9 +18,14 @@
         - Additional information about change #1
 -->
 ## Current Release
+<<<<<<< HEAD
```
Comment (markcowl) [inline src/ResourceManager/TrafficManager/Commands.TrafficManager2/ChangeLog.md]: merge issue

File: src/ResourceManager/UsageAggregates/Commands.UsageAggregates/ChangeLog.md [hunk 21]
```diff
@@ -18,9 +18,14 @@
         - Additional information about change #1
 -->
 ## Current Release
+<<<<<<< HEAD
```
Comment (markcowl) [inline src/ResourceManager/UsageAggregates/Commands.UsageAggregates/ChangeLog.md]: merge issue

File: src/ResourceManager/Websites/Commands.Websites/ChangeLog.md [hunk 22]
```diff
@@ -18,9 +18,14 @@
         - Additional information about change #1
 -->
 ## Current Release
+<<<<<<< HEAD
```
Comment (markcowl) [inline src/ResourceManager/Websites/Commands.Websites/ChangeLog.md]: merge issue

File: src/ResourceManager/Automation/Commands.Automation.Test/Commands.Automation.Test.csproj (hunk 1 above)
Comment (cormacpayne) [reply src/ResourceManager/Automation/Commands.Automation.Test/Commands.Automation.Test.csproj]: Discussed offline -- these references were accidentally added in previous PRs and can safely be removed

**Comment by markcowl** [comment]
Ond emand run here: https://azuresdkci.westus2.cloudapp.azure.com/view/PowerShell/job/powershell-demand/1014/

**Pull request merged**